Civi/Install/Requirements.php

```diff
@@ -227,8 +227,7 @@ public function checkMysqlConnection(array $db_config) {
       'details' => "Connected",
     );
 
-    $host = isset($db_config['server']) ? $db_config['server'] : $db_config['host'];
-    $conn = @mysqli_connect($host, $db_config['username'], $db_config['password']);
+    $conn = @mysqli_connect($host, $db_config['username'], $db_config['password'], $db_config['database'], $db_config['port']);
```

Comment:
It must have been $db_config['host']. But it worked while I tested because, I think DB was in localhost. 

I will correct the problem and resubmit